src/index.js

```diff
@@ -10,7 +10,10 @@ import ActivityRepository from './ActivityRepository';
 import HydrationRepository from './HydrationRepository';
 import SleepRepository from './SleepRepository';
 
-import userData from './data/users';
+// import userData from './data/users';
+var userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
```

Comment:
It's great to read through this PR because learning about fetch calls in class lets me implement my knowledge of it and the capability to understand what I'm reading.